examples/structured_data/classification_with_tfdf.py

```diff
@@ -126,14 +115,39 @@
     CATEGORICAL_FEATURES_WITH_VOCABULARY.keys()
 )
 
+"""
+Now we perform basic data preparation.
+"""
+
+
+def prepare_dataframe(dataframe):
+    # Convert the target labels from string to integer.
+    dataframe[TARGET_COLUMN_NAME] = dataframe[TARGET_COLUMN_NAME].map(
+        TARGET_LABELS.index
+    )
+    # Cast the categorical features to string.
+    for feature_name in CATEGORICAL_FEATURES_WITH_VOCABULARY:
+        dataframe[feature_name] = dataframe[feature_name].astype(str)
+
+
+prepare_dataframe(train_data)
+prepare_dataframe(test_data)
+
+"""
+Now let's show the shapes of the training and test dataframes, and display some instances.
+"""
+
+print(f"Train data shape: {train_data.shape}")
+print(f"Test data shape: {test_data.shape}")
+print(train_data.head().T)
+
 """
 ## Configure hyperparameters
 
 You can find all the parameters of the Gradient Boosted Tree model in the
 [documentation](https://www.tensorflow.org/decision_forests/api_docs/python/tfdf/keras/GradientBoostedTreesModel)
 """
 
-GROWING_STRATEGY = "BEST_FIRST_GLOBAL"
 NUM_TREES = 250
```

Comment:
Please remove.
See comment in https://github.com/keras-team/keras-io/pull/782

Reply:
I just wanted to expose some of the key parameters of the algorithms to be visible to the readers. I can add descriptions as comments to each one. Also happy to remove them. What do you think?

Reply:
Do you think it is better to use the [documentation](https://www.tensorflow.org/decision_forests/api_docs/python/tfdf/keras/GradientBoostedTreesModel) link instead? (next to the model construction object)

Reply:
Yes it is equally good.

Note that this documentation pages also list all the Keras model arguments (not just the hyper-parameters) and it does not list the possible values the say way.

Reply:
Done